src/components/Avatar/Avatar.scss

```diff
@@ -1,20 +1,17 @@
 @import "~styles/main";
 
-.container {
+.dotted {
   img {
```

Comment:
does it adjust all sizes of images?

Reply:
![image](https://user-images.githubusercontent.com/39233007/121918368-b48d3780-cd55-11eb-9a02-feb1b5b03bb2.png)
Made images look like this